### DISCUSSION TITLE
Keyset Pagination supported for repository_tree interface?

### DISCUSSION THREAD

**Discussion by AnansiRafa** [python-gitlab/python-gitlab#1902] [Q&A] [answered]
GitLab docs state that [number-based pagination](https://docs.gitlab.com/ee/api/repositories.html#list-repository-tree) for their repository tree endpoint will be deprecated in version `15.0`, apparently to be released in May. Does this wrapper support keyset pagination for this interface?

Thanks!

**Comment by JohnVillalovos** [comment, marked as answer]
If using `project.repository_tree(as_list=False)` then it should support that.

https://github.com/python-gitlab/python-gitlab/blob/bd1ecdd5ad654b01b34e7a7a96821cc280b3ca67/gitlab/client.py#L991-L999
